aten/src/ATen/native/cuda/AbsKernel.cu

```diff
@@ -0,0 +1,40 @@
+#include <ATen/native/UnaryOps.h>
+#include <ATen/native/cuda/Loops.cuh>
+#include <ATen/Dispatch.h>
+#include <ATen/native/DispatchStub.h>
+#include <ATen/native/TensorIterator.h>
+
+namespace at { namespace native {
+
+// We manually overload abs because std::abs does not work with thrust::complex types and ROCm.
```

Comment:
Nit: I forgot to update this in my original PR. Should be:
```
// We manually overload abs because std::abs does not work with ROCm.
```
We are no longer using `thrust::complex` any more